website/docs/docs/collaborate/explore-projects.md

```diff
@@ -51,75 +47,107 @@ By default, dbt Explorer shows the project's [applied state](/docs/dbt-cloud-api
 
 To explore the lineage graphs of tests and macros, view [their resource details pages](#view-resource-details). By default, dbt Explorer excludes these resources from the full lineage graph unless a search query returns them as results.
 
-To interact with the full lineage graph, you can:
+<expandable alt_header="How can I interact with the full lineage graph?">
 
 - Hover over any item in the graph to display the resource’s name and type.
 - Zoom in and out on the graph by mouse-scrolling.
 - Grab and move the graph and the nodes.
 - Right click on a node (context menu) to:
-    - Refocus on the node, including its parent and child nodes
-    - Refocus on the node and its children only
-    - Refocus on the node and it parents only
+    - Refocus on the node, including its upstream and downstream nodes
+    - Refocus on the node and its downstream nodes only
+    - Refocus on the node and it upstream nodes only
     - View the node's [resource details](#view-resource-details) page
-
 - Select a resource to highlight its relationship with other resources in your project. A panel opens on the graph’s right-hand side that displays a high-level summary of the resource’s details. The side panel includes a **General** tab for information like description, materialized type, and other details.
     - Click the Share icon in the side panel to copy the graph’s link to your clipboard.
     - Click the View Resource icon in the side panel to [view the resource details](#view-resource-details). 
-- [Search and select specific resources](#search-resources) or a subset of the DAG using [selectors](/reference/node-selection/methods) and [graph operators](/reference/node-selection/graph-operators). This can help you narrow the focus on the resources that interest you. For example:
-    - `+[RESOURCE_NAME]` &mdash; Displays all parent nodes of the resource
-    - `resource_type:model [RESOURCE_NAME]` &mdash; Displays all models matching the name search
+- Use [selectors](/reference/node-selection/methods) (in the search bar) to select specific resources or a subset of the DAG. This can help narrow the focus on the resources that interest you. All selectors are available for use, except those requiring a state comparison (result, source status, and state). You can also use the `--exclude` and the `--select` flag (which is optional). Examples:
```

Comment:
Could potentially start the section with 
In the search bar:
- Use
- Use
- Use

so you don't have to clarify on each step.

Reply:
good thought! i played around with that construction too but it looked a bit awkward as well